docs/how-to-guides/block-tutorial/block-controls-toolbar-and-sidebar.md

```diff
@@ -192,17 +182,100 @@ registerBlockType( 'gutenberg-examples/example-04-controls-esnext', {
 
 Note that `BlockControls` is only visible when the block is currently selected and in visual editing mode. `BlockControls` are not shown when editing a block in HTML editing mode.
 
-## Inspector
+## Sidebar Inspector
 
 ![Screenshot of the inspector panel focused on the settings for a Paragraph block](https://raw.githubusercontent.com/WordPress/gutenberg/HEAD/docs/assets/inspector.png)
 
-The Settings Sidebar is used to display less-often-used settings or settings that require more screen space. The Settings Sidebar should be used for **block-level settings only**.
+The Sidebar Inspector is used to display less-often-used settings or settings that require more screen space. The Settings Sidebar should be used for **block-level settings only**.
```

Comment:
```suggestion
The Settings Sidebar is used to display less-often-used settings or settings that require more screen space. The Settings Sidebar should be used for **block-level settings only**.
```

Per the [Glossary](https://developer.wordpress.org/block-editor/handbook/glossary/) the sidebar is referred to as the Settings Sidebar so we should be consistent with the name throughout.